_posts/2019-02-13-extending-pg-search-sort.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+logo: "--white"
+header_style: "c-header--black"
+title: "Extending sort capabilities of pg_search"
+date: "Feb 13, 2019"
+author: "Roel Bondoc"
+author_role: "Fullstack Developer Razeware"
+author_bio: "Interests: Ruby on Rails, games, and basketball."
+author_image: "roel-bondoc@2x.jpeg"
+color: "#ffcb32"
+hero: "c-post-hero--dark"
+image: "/assets/img/pattern-6@2x.png"
+category: "development"
+excerpt: "Adjusting the sort capabilities pg_search to provide better search results."
+---
+
+The search functionality of raywenderlich.com is powered by Postgresql using the `pg_search` ruby gem. Out of the box it is a fantastic library and makes building a search feature easy. Although, as easy as it was, we were quick to discover that our search results were not returning desired results. So I decided to take a closer look at how `pg_search` constructs its queries.
```

Comment:
- PostgreSQL.
- Before talking about the gem, I think you should give a sentence about the fact that PostgreSQL has some high-powered search functionality. That for simple search tasks you might not need to reach for ES. That'll put the gem in context—we want to access this functionality, but want to use a nice ruby API instead of messing about with SQL.
- Add a link to the gem.